wiki/Live_streaming_osu!/en.md

```diff
@@ -1,37 +1,36 @@
-If you are visiting this page, then it is likely you have an interest and want to learn how to live stream [osu!](osu! "wikilink") on twitch.tv. Doing so is a very simple process but you may require a reliable computer to be able to run both osu! and send live encoding to [twitch.tv](http://www.twitch.tv) (which from now on will be referred to as TTV) at the same time.
+# Live streaming osu!
 
-Step 1 - Registration
-=====================
+If you are visiting this page, then it is likely you have an interest and want to learn how to live stream *osu!* on twitch.tv. Doing so is a very simple process but you may require a reliable computer to be able to run both osu! and send live encoding to [twitch.tv](https://www.twitch.tv) (which from now on will be referred to as TTV) at the same time.
```

Comment:
❗️ but... what if I want to use, let's say, YouTube Gaming or other services? :p
(Just leave as-is right now)